contentcuration/contentcuration/static/js/edit_channel/share/hbtemplates/share_editor_item.handlebars

```diff
@@ -1,9 +1,18 @@
-<span class="editor_name truncate" title="{{editor.first_name}} {{editor.last_name}}">{{editor.first_name}} {{editor.last_name}}</span>
-<span class="editor_email">&nbsp;(<div class="share_item_email truncate" title="{{editor.email}}">{{editor.email}}</div>{{#if isviewonly}} - {{formatMessage (intlGet 'messages.view_only')}}{{/if}})</span>
-<div class="pull-right">
+<div class="row">
+<div class="field col-xs-4 truncate">
```

Comment:
The `truncate` class will cut any overflowing text and add an ellipsis. I switched to the div + bootstrap grid system to handle the sizing more consistently